src/sgame/sg_admin.cpp

```diff
@@ -2014,7 +2012,7 @@ bool G_admin_readconfig( gentity_t *ent )
 			{
 				admin_readconfig_string( &cnf, l->name, sizeof( l->name ) );
 				// max printable name length for formatting
-				len = Color::StrlenNocolor( l->name );
+				len = strlen( l->name );
```

Comment:
Why change this?

Reply:
Already explained in the initial comment, the previous code made no sense as it was adding the length of the decoloured level name in the listplayers entry to `admin_level_maxname` resulting in a double-length alignment in some cases; that code never worked properly.

I changed this because in every instance `admin_level_maxname` was used, it was being compared to another decoloured admin level name which is essentially a double negative. For formatting, this makes the most sense anyway as you'd never not want to display coloured admin levels in any relevant formatting.

The only two places this is used is `listplayers` and `listadmins`, where I have fixed the same issue in both commands with this PR.

Reply:
The original code is right here. The change only appears to work because all the names in the examples are ASCII-only and have the same number of characters in color tokens.